atomic_reactor/plugins/exit_remove_worker_metadata.py

```diff
@@ -31,7 +31,11 @@ def run(self):
 
         for platform, build_annotations in worker_builds.items():
             try:
-                cm_key, _ = get_platform_config(platform, build_annotations)
+                if ('metadata_fragment' in build_annotations and
+                        'metadata_fragment_key' in build_annotations):
+                    cm_key, _ = get_platform_config(platform, build_annotations)
+                else:
+                    continue
```

Comment:
Wouldn't need an `else` keyword if the condition was reversed:
```
if ('metadata_fragment' not in build_annotations or
        'metadata_fragment_key' not in build_annotations):
    continue

cm_key, _ = ...
```

Reply:
changed